rero_ils/modules/holdings/api_views.py

```diff
@@ -139,3 +149,98 @@ def receive_regular_issue(holding_pid):
         abort(400)
     # the created item of type issue is returned
     return jsonify({'issue': issue})
+
+
+def do_holding_jsonify_action(func):
+    """Jsonify loan actions for holdings methods.
+
+    This method for the circulation actions that required access to the holding
+    object before executing the invenio-circulation logic.
+    """
+    @wraps(func)
+    def decorated_view(*args, **kwargs):
+        try:
+            data = flask_request.get_json()
+            if not data.get('description'):
+                raise MissingRequiredParameterError
```

Comment:
```python
try:
   description = data.pop('description')
except KeyError as ke:
   raise MissingRequiredParameterError(ke.message) 
````
You can use `description` variable into `item_metadata` dict.
You can remove line `data.pop('description', None)`(L186)